samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnClusterResourceManager.java

```diff
@@ -575,6 +576,17 @@ public void onStopContainerError(ContainerId containerId, Throwable t) {
     }
   }
 
+  @Override
+  public boolean isResourceExpired(SamzaResource resource) {
+    /**
```

Comment:
Use `//` for comments inside methods, not `/*`. IntelliJ warns about dangling javadocs for these.